rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcCustomServer.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.rpc.netty;
+
+import co.rsk.config.RskSystemProperties;
+import co.rsk.rpc.ModuleDescription;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.googlecode.jsonrpc4j.*;
+
+import io.netty.channel.ChannelHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.*;
+
+import static com.googlecode.jsonrpc4j.ErrorResolver.JsonError.INTERNAL_ERROR;
+
+@ChannelHandler.Sharable
+public class JsonRpcCustomServer extends JsonRpcBasicServer {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger("JsonRpcCustomServer");
+    private final RskSystemProperties rskSystemProperties;
```

Comment:
I think it would be better to only include the required properties modules and the RpcTImeout and leave the RskSystemProperties class independent from this object.

Reply:
Done